papis_zotero/sql.py

```diff
@@ -117,8 +115,7 @@ def get_creators(connection: sqlite3.Connection,
 ZOTERO_QUERY_ITEM_ATTACHMENTS = """
 SELECT
     items.key,
-    itemAttachments.path,
-    itemAttachments.contentType
+    itemAttachments.path
```

Comment:
Why was the content type removed? It seems to be used in the query `WHERE` statement below.

Reply:
You're right. I'll revert this deletion.